dashboard/apps/prototype/models.py

```diff
@@ -212,7 +227,61 @@ def money_spent(self, start_date, end_date, contractor_only=False,
             tasks = tasks.all()
         spending_per_task = [task.money_spent(start_date, end_date)
                              for task in tasks]
-        return sum(spending_per_task)
+        costs = self.cost_between(start_date, end_date)
+        return sum(spending_per_task) + costs
+
+    def cost_between(self, start_date, end_date):
+        def cost_of_cost(cost):
+            return cost.cost_between(start_date, end_date)
+
+        return sum(map(cost_of_cost, self.costs.filter(
+            Q(end_date__lte=end_date) | Q(end_date__isnull=True),
+            start_date__gte=start_date
+        ))) or Decimal('0')
+
+
+class Cost(models.Model):
```

Comment:
Is it worth renaming this to something that makes it obvious it's non-staff costs?